packages/e2e-tests/specs/performance/i18n-filters.test.js

```diff
@@ -0,0 +1,291 @@
+/**
```

Comment:
@youknowriad – do you think it's something we want to keep? I guess it's fine to have it to prevent regressions. It might be good to extract some common logic though. It looks like performance tests take over 30 minutes to run now.